### PR TITLE
Super Admin workflow definitions

### PR DESCRIPTION
**Pull request by davidmoore1**

### CODE REVIEW

File: source/SIL.AppBuilder.Portal.Frontend/src/data/containers/resources/workflow-definition/with-data-actions.tsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+interface IOwnProps {
+  workflowDefinition: WorkflowDefinitionResource;
+}
+
+
+type IProps =
+  & IOwnProps
+  & WithDataProps;
+
+const mapRecordsToProps = (passedProps) => {
```
Comment (giancorzo) [inline source/SIL.AppBuilder.Portal.Frontend/src/data/containers/resources/workflow-definition/with-data-actions.tsx]: this is not doing anything

File: source/SIL.AppBuilder.Portal.Frontend/src/data/containers/resources/workflow-definition/with-data-actions.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        updateStoreType: this.updateStoreType
+      };
+
+      return <WrappedComponent {...this.props} {...actionProps} />;
+    }
+
+  }
+
+  return compose(
+    withOrbit(mapRecordsToProps)
```
Comment (giancorzo) [inline source/SIL.AppBuilder.Portal.Frontend/src/data/containers/resources/workflow-definition/with-data-actions.tsx]: if you remove the mapRecordsToProps replace this part with withOrbit({})

File: source/SIL.AppBuilder.Portal.Frontend/src/data/models/workflow-definition.ts [hunk 3]
```diff
@@ -1,13 +1,15 @@
-import { AttributesObject, ResourceObject } from 'jsonapi-typescript';
+import { AttributesObject, ResourceObject } from "jsonapi-typescript";
```
Comment (giancorzo) [inline source/SIL.AppBuilder.Portal.Frontend/src/data/models/workflow-definition.ts]: dont use " better use '

File: source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/admin/settings/index.tsx [hunk 4]
```diff
@@ -5,14 +5,18 @@ import { withAdminLayout } from '@ui/components/layout/admin';
 
 import { Switch, Route } from 'react-router-dom';
 import OrganizationsRoute from '@ui/routes/admin/settings/organizations';
+import WorkflowDefinitionsRoute from '@ui/routes/admin/settings/workflow-definitions';
 
 export const pathName = '/admin/settings';
+export const workflowDefinitionsPathName = '/admin/settings/workflow-definitions';
```
Comment (giancorzo) [inline source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/admin/settings/index.tsx]: this doesn't need to be exported and should be in the inner component.
if you are not going use pathName remove it

File: source/SIL.AppBuilder.Portal.Frontend/src/data/models/workflow-definition.ts (hunk 3 above)
Comment (NullVoxPopuli) [reply source/SIL.AppBuilder.Portal.Frontend/src/data/models/workflow-definition.ts]: the linter should catch this. No need to be a human linter :)

File: source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/admin/settings/workflow-definitions/list/index.tsx [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+      <>
+        <h2 className='sub-page-heading'>
+          {t('admin.settings.workflowDefinitions.title')}
+        </h2>
+        <div className='m-b-xxl'>
+          <button
+            className='ui button tertiary uppercase large m-b-lg'
+            onClick={this.showAddForm}
+          >
+            Add Workflow Definition
```
Comment (giancorzo) [inline source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/admin/settings/workflow-definitions/list/index.tsx]: translation missing

File: source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/admin/settings/workflow-definitions/common/form.tsx [hunk 6]
```diff
@@ -0,0 +1,275 @@
...
+        },{
+          storeType
+        });
+      } catch(e) {
+        toast.error(e);
+      }
+    }
+  }
+
+  cleanForm = () => {
```
Comment (giancorzo) [inline source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/admin/settings/workflow-definitions/common/form.tsx]: this is not been used. You can remove it

Comment (davidmoore1) [reply source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/admin/settings/workflow-definitions/common/form.tsx]: Removing it and the one in organization as well

File: source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/admin/settings/workflow-definitions/list/index.tsx (hunk 5 above)
Comment (davidmoore1) [reply source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/admin/settings/workflow-definitions/list/index.tsx]: I noticed this one when I first entered it and then couldn't find it anymore.  Thought I must have already fixed it.

File: source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/admin/settings/workflow-definitions/common/form.tsx (hunk 6 above)
Comment (davidmoore1) [reply source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/admin/settings/workflow-definitions/common/form.tsx]: Removed this one and the organization version which was also not being called

**Pull request merged**